src/rdb-cli/Makefile

```diff
@@ -0,0 +1,14 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = rdb-cli
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	rm -rf ./librdb
+	git clone https://github.com/redis/librdb.git
+	pushd ./librdb/
+	git checkout 2fdfc0c2bc914d643fe3f86e6715aeb843d8966e
```

Comment:
Is this line really needed? It's not tagged or anything.

Reply:
Currently, the librdb doesn't have any version tag. So, it will have risks if the SONiC always get its latest version. I checkout this commit because I have tested this commit and want to avoid the incompatibility caused by the future latest commit. I have already opened an issue below to librdb for improvement.
https://github.com/redis/librdb/issues/58
If you have a better suggestion, please share it with me. Thanks.

Reply:
I guess this is fine. It's worth noting that for the purposes of the builds that happen as part of PR/commit checks, there is a version pinning infra that will make sure this git hash is used. However, those version pinning files will get updated to the latest version if the build with the latest version of everything passes, and since we don't have runtime checks here, it probably is indeed better to have `git checkout` here.

Reply:
Thanks.